Dashboard release (#2229)
SumoLogic/sumologic-documentation: blog-service/2023-04-28-dashboard.md

---
title: April 28, 2023 (Dashboards)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - dashboards new
  - dashboard migration
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> We're excited to announce the release of several new features in the **Dashboards (New)**:

Reviewer: We should add how these features will benefit the user and where they can find them.

---
title: April 28, 2023 (Dashboards)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - dashboards new
  - dashboard migration
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

We're excited to announce the release of several new features in the **Dashboards (New)**:

> - **Sankey Chart**. You can find our new **Sankey diagram** feature under the **Categorical** panel type, which is built using two fields in your query, such as `fromstate` and `tostate`. The Sankey diagram helps you to understand the flow of the log events within a distributed system, and the width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.

Reviewer 2: ```suggestion
- **Sankey Chart**. Our new **Sankey diagram** feature, which can you find under the **Categorical** panel type, is built using two fields in your query, such as `fromstate` and `tostate`. The Sankey diagram helps you understand the flow of the log events within a distributed system. The width of an arrow or stripe in the Sankey diagram shows the proportion of a quantity.
```

> 
> - **Missile Map**. You can find our new **Missile Map** feature under the **Map** panel type, which helps you to get visibility into the geographic origins of threats and their target points, by hovering over each line, you will be able to find the threat name, latitude and longitude details of threat origin, and latitude and longitude details of threat target point.
> 

Reviewer 2: ```suggestion
- **Missile Map**. Our new **Missile Map** feature, which you can find under the **Map** panel type, provides visibility into the geographic origins of threats and their target points. By hovering over each line, you can find the threat name, latitude/longitude details of threat origin, and latitude/longitude details of threat target point.
```
